tests/test_interfaces.py

```diff
@@ -36,10 +36,25 @@ def test_interfaces(duthosts, enum_frontend_dut_hostname, tbinfo, enum_asic_inde
     verify_ip_address(host_facts, mg_facts['minigraph_vlan_interfaces'])
     verify_ip_address(host_facts, mg_facts['minigraph_lo_interfaces'])
 
+    router_mac = duthost.facts['router_mac']
+    verify_mac_address(host_facts, mg_facts['minigraph_portchannel_interfaces'], router_mac)
+    verify_mac_address(host_facts, mg_facts['minigraph_vlan_interfaces'], router_mac)
+    verify_mac_address(host_facts, mg_facts['minigraph_interfaces'], router_mac)
+
 def verify_port(host_facts, ports):
     for port in ports:
         pytest_assert(host_facts[port]['active'], "interface {} is not active".format(port))
 
+def verify_mac_address(host_facts, intfs, router_mac):
+    for intf in intfs:
+        if intf.has_key('attachto'):
```

Comment:
```suggestion
        if 'attachto' in intf:
```
Suggest using the `in` syntax because it is compatible with both python2 and python3.